src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -72,6 +72,11 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "DELETE" }
     ),
+    deleteUserAccountLoader: gravityLoader(
+      (id) => `user/${id}`,
```

Comment:
So this is actually `me` not the admin URL

Reply:
https://github.com/artsy/gravity/blob/d53fd1a51711b98103feabdc64503be9d893b585/app/api/v1/me_endpoint.rb#L96-L102